RELEASE.md

```diff
@@ -0,0 +1,9 @@
+# OpenShift Knative Serving Release procedure
+
+The OpenShift Knative Serving release cut is mostly automated and requires only two manual steps for enabling the CI runs on the `openshift/release` repository.
+
+No manual creation of a midstream `release-v1.x` branch is needed. The nightly Jenkins job, does create a `release` branch, as soo as the upstream has created a new release tag. The code for this script is located in this [script](./openshift/release/mirror-upstream-branches.sh), which does mirror the upstream release tag to our midstream `release` branches.
```

Comment:
Release tags upstream have a patch version. 
I suspect we will use `release-v1.x` and ignore the patch version? For example upstream knative-v1.4.1 is created and we create downstream knative-v1.4? If knative-v1.4.2 comes out we override the old one?  
Reason I am asking this is that at the last release meeting we were inclining to keep only major.minor for release branches, in order to align with Eventing.

Reply:
Yes, we will use `release-v1.x` and ignore the patch version. For the patch version, we need to manually pick it and backport.